src/metatensor/models/share/metatensor-models-completion.bash

```diff
@@ -0,0 +1,96 @@
+_metatensor-models()
```

Comment:
Yes, there are but I didn't found how they can handle file types, which are quite important to reduce errors especially since we have this probable confusion with .pt and .ckpt.

Reply:
fair enough!